packages/smooth_app/lib/pages/prices/prices_stats_page.dart

```diff
@@ -157,315 +71,267 @@ class _PricesStatsPageState extends State<PricesStatsPage> {
+          Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Center(
               child: Text(
-                '${localizations.prices_stats_last_updated} ${stats.lastUpdated}',
-                style: Theme.of(context)
-                    .textTheme
-                    .bodySmall
-                    ?.copyWith(color: subtitleColor),
+                '${localizations.prices_stats_last_updated} ${_formatDateTime(statsData!['updated'].toString())}',
+                style: Theme.of(context).textTheme.bodySmall,
               ),
             ),
-            const SizedBox(height: 16),
-          ],
-        ),
+          ),
+      ],
+    );
+  }
+
+  Widget _getSectionHeader(IconData iconData, String description) {
+    return ListTile(
+      leading: Icon(iconData),
+      title: Text(
+        description,
+        style: const TextStyle(fontWeight: FontWeight.bold),
       ),
+      contentPadding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
     );
   }
 
-  Widget _buildListItems(List<List<dynamic>> items) {
-    return ListView.separated(
-      shrinkWrap: true,
-      physics: const NeverScrollableScrollPhysics(),
-      itemCount: items.length,
-      separatorBuilder: (BuildContext context, int index) =>
-          const SizedBox(height: 8),
-      itemBuilder: (BuildContext context, int index) {
-        final List<dynamic> item = items[index];
-        final String number = item[0] as String;
-        final String label = item[1] as String;
-        final IconData icon = item[2] as IconData;
-        final bool showRedirectArrow = item.length > 3 && item[3] == true;
-        final String? url = item.length > 4 ? (item[4] as String) : null;
-        return StatsListTile(
-          number: number,
-          label: label,
-          icon: icon,
-          showRedirectArrow: showRedirectArrow,
-          onTap: url != null ? () => LaunchUrlHelper.launchURL(url) : null,
-        );
-      },
+  Widget _getDataTile({
+    required int? value,
+    int? denominator,
+    required String description,
+    String? url,
+  }) {
+    if (value == null) {
+      return emptyWidget;
+    }
+
+    final String displayValue = denominator == null
+        ? value.toString()
+        : '$value / $denominator';
+
+    return ListTile(
+      title: Text(displayValue),
+      subtitle: Text(description),
+      trailing: url == null ? null : const Icon(Icons.open_in_new),
+      onTap: url == null ? null : () => LaunchUrlHelper.launchURL(url),
+      dense: true,
+      contentPadding: const EdgeInsets.symmetric(horizontal: 24.0),
     );
   }
-}
+
+  int? _getValue(
+      final Map<String, dynamic> json,
+      final String tag,
+      ) =>
+      json[tag] as int?;
+
+
+  String _formatDateTime(String dateTimeStr) {
+    try {
+      final DateTime dateTime = DateTime.parse(dateTimeStr);
+      return DateFormat('d MMMM yyyy at HH:mm:ss').format(dateTime);
```

Comment:
It's still in English (" at ", no locale specified)
I would recommend using something like `DateFormat.yMd(ProductQuery.getLanguage().offTag).add_jms()`.